allennlp/__main__.py

```diff
@@ -0,0 +1,26 @@
+from typing import Sequence
+import argparse
+import logging
+import sys
+
+from allennlp.commands.bulk import add_bulk_subparser
+from allennlp.common.params import PARAMETER
+
+# disable parameter logging
+logging.disable(PARAMETER)
```

Comment:
Why did you end up adding this?  What did it fix for you?  It did not fix my output from flask.